### PR TITLE
Landing page for admins

### LINKED ISSUE #73: Admin landing page
We need a simple landing page for admins so that they do not need to navigate the site's admin features via URLs.

### PR DESCRIPTION
**Pull request by toni-santos**
Closes #73

### CODE REVIEW

File: resources/js/Pages/Admin.vue [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    },
+    "admin.departments.index": {
+        label: "Departments",
+    },
+    "admin.stands.index": {
+        label: "Stands",
+    },
+    "admin.eventDays.index": {
+        label: "Event Days",
+    },
```
Comment (ttoino) [inline resources/js/Pages/Admin.vue]: These should probably be in portuguese

File: routes/web.php [hunk 2]
```diff
@@ -61,6 +61,10 @@ function () {
                     '/stands' => StandCRUDController::class,
                     '/eventDays' => EventDayCRUDController::class,
                 ]);
+
+                Route::name('index')->get('/index', function () {
```
Comment (ttoino) [inline routes/web.php]: I'd prefer `/`

**Review by ttoino — CHANGES_REQUESTED** [review]

**Comment by pedrosilva17** [comment]
Agree with João's observations. Other than that, useful landing page 👍

**Comment by pedrosilva17** [comment]
I just remembered, maybe we should have a section for managing enrollments? We could change the amount of points of a participant in case a problem arises, just in case

**Review by ttoino — APPROVED** [review]

**Review by Naapperas — APPROVED** [review]

**Pull request merged**